inductiva/tasks/task.py

```diff
@@ -231,7 +231,7 @@ def download_outputs(
                 files are downloaded.
             output_dir: Directory where to download the files. If None, the
                 files are downloaded to the default directory. The default is
-                {inductiva.working_dir}/{inductiva.output_dir}/{task_id}.
+                {inductiva._working_dir}/{inductiva._output_dir}/{task_id}.
```

Comment:
The default should be:

```suggestion
                {inductiva.get_output_dir()}/{output_dir}/{task_id}
```

if `output_dir` is None, files are written to:

```suggestion
                {inductiva.get_output_dir()}/{task_id}
```

if `inductiva.get_output_dir() is None` and `output_dir is not None`:

```suggestion
                ./{output_dir}/{task_id}
```

if bot are None:

```suggestion
                ./{task_id}
```